apps/frontend/src/components/RequestModal/RequestModal.tsx

```diff
@@ -18,13 +19,97 @@ interface Props {
 
 export default function RequestModal({ show, handleCloseModal, request }: Props) {
   const [loading, setIsLoading] = useState(false);
+  const [validated, setValidated] = useState(false);
+  const [showForm, setShowForm] = useState(false);
   const revalidator = useRevalidator();
+  const { id: neighborhoodId } = useParams();
+  const submit = useSubmit();
+
+  const user = localStorage.getItem('user');
+  const { username, id: userId } = user ? JSON.parse(user) : null;
+
+  const validateTextArea = (event: FocusEvent<HTMLTextAreaElement>) => {
+    const textarea = event.currentTarget as HTMLTextAreaElement;
+    const validPattern = /\s*(\S\s*){4,}/;
+
+    if (!validPattern.test(textarea.value)) {
+      textarea.setCustomValidity('The content needs to be at least 4 characters long.');
+    } else {
+      textarea.setCustomValidity('');
+    }
+
+    textarea.reportValidity();
+  };
+
+  const hideValidation = (event: ChangeEvent<HTMLTextAreaElement>) =>
```

Comment:
Something I was trying to figure out while working on this too, was hiding the validation error when we click outside of the textarea. It's really annoying trying to do something else and having this error message pop up.

Reply:
The thing is I've set this up in the exact opposite way: to pop up when the focus moves outside the textarea. Ideally, we only want it to show when we attempt to submit the form.

Reply:
I'm investigating a possible solution using refs. Will keep you updated.

Reply:
I think I've managed to fix it! Please let me know if it works on your end.

Reply:
Awesome! It did work but with some minor issues. I'm going to push some changes, and let me know if it still works for you, too.